server/service/osquery.go

```diff
@@ -374,6 +374,8 @@ func (svc *Service) GetClientConfig(ctx context.Context) (map[string]interface{}
 	}
```

Comment:
`svc.ds.ListPacksForHost` should return only "user packs" now, aka 2017 packs (basically those with `packs.pack_type IS NULL`).

Reply:
If we do the above, do we need the below exclusion mechanism?...

Reply:
Currently, there are three kind of (internal) packs in Fleet:
1. `packs.pack_type == "global"` are used to group the global scheduled queries.
2. `packs.pack_type == "team-$ID"` are used to group the team scheduled queries.
3. `packs.pack_type == NULL` are the 2017 packs.

The way I was seeing is:
In the data migration, (1) and (2) will be removed from the `scheduled_queries` and migrated to `queries` table.
(3) we have to leave untouched to support and not break 2017 packs.

If we do the above, then we only have to change `ListPacksForHost` to return the 2017 packs and then we don't need any exclusion logic AFAICS.

Let me know if we are on same page there.

Reply:
That makes sense - thanks Lucas

Reply:
Done in https://github.com/fleetdm/fleet/pull/12723/commits/3385819b60a3cd87305cba6c372c65e9eb078871